README.md

```diff
@@ -0,0 +1,69 @@
+# OCP DNSNameResolver
```

Comment:
Out of curiosity: why OCP and not OpneShift?

Reply:
I see. However OCP is a product name which can be less known compared to OpenShift. It could `openshift_dnsnameresolver` to follow the same naming practices.

Reply:
Ack. I'll update the Readme.

Reply:
Just want to clarify whether you want me to mention OpenShift only in the Readme file or do you want me to change the plugin name as well. The plugin name was discussed during the EP phase and that's what was accepted by both the SDN and the NE team at that time. The plugin name is mentioned in the EP: https://github.com/openshift/enhancements/pull/1335/files